src/image.rs

```diff
@@ -113,34 +113,34 @@ pub type ImageResult<T> = Result<T, ImageError>;
 #[derive(Clone, Copy, PartialEq, Eq, Debug)]
 pub enum ImageFormat {
     /// An Image in PNG Format
-    PNG,
+    Png,
 
     /// An Image in JPEG Format
-    JPEG,
+    Jpeg,
 
     /// An Image in GIF Format
-    GIF,
+    Gif,
 
     /// An Image in WEBP Format
-    WEBP,
+    Webp,
```

Comment:
An alternate spelling would be `WebP`, the official capitalization.

Reply:
I have no strong opinion, but `Webp` is fine.